admin-guide.md

```diff
@@ -270,3 +277,19 @@ will be captured. Then `"metrics_calculate"` will be called with the
 `"output_metrics"` as the first argument and the file generated in the previous
 step as the second argument. If it exits with a non-zero return code, then the
 test is allowed to pass.
+
+## Testing Víðarr Locally
+
+To test Víðarr locally, ensure that you satisfy these requirements:
+
+- Make sure the .sh file created from the instructions provided in [this step](https://github.com/oicr-gsi/vidarr/blob/master/admin-guide.md#creating-a-development-environment) points to the jars you created [here](https://github.com/oicr-gsi/vidarr/blob/master/admin-guide.md#installing-v%C3%AD%C3%B0arr-on-a-linux-instance).
```

Comment:
I'd be more specific here and replace "this step" with "the Creating a Development Environment" step (the link is good though. Same idea for the other linked steps